packages/core/src/InputGroup/InputGroup.js

```diff
@@ -18,7 +21,7 @@ const InputGroup = styled.div`
     flex: 1 1 auto;
   }
 
-  & ${Input} {
+  & ${StyledInput} {
```

Comment:
could we change this to `& input` since it'll still select the input element?

Reply:
Huh good question, I can try that and see what happens.